src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,78 @@
+import cn from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  selectedPerson: Person | null;
+  isDropdownActive: boolean;
+  query: string;
+  onQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onPersonClick: (person: Person) => void;
+  onFocus: () => void;
+  onBlur: () => void;
+};
+
+export const Autocomplete: React.FC<Props> = ({
+  people,
+  selectedPerson,
+  isDropdownActive,
+  query,
+  onQueryChange,
+  onPersonClick,
+  onFocus,
+  onBlur,
+}) => {
+  return (
+    <>
+      <h1 className="title">
+        {!selectedPerson
+          ? 'No person selected'
+          : `${selectedPerson?.name} (${selectedPerson?.born} = ${selectedPerson?.died})`}
```

Comment:
```suggestion
        {!selectedPerson
          ? 'No person selected'
          : `${selectedPerson.name} (${selectedPerson.born} = ${selectedPerson?.died})`}
```
you already checked if selectedPerson exists